### PR TITLE
feat: add blog post on arcus scripting v0.6

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds a blog post on some of the new features of the Arcus Scripting v0.6 release.

Relates to #203

### CODE REVIEW

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6
+Recently, we have released Arcus Scripting v0.6. The scripting library was quiet for a while, without any updates, so let's change that with this run-down post of new functionality coming to Arcus Scripting.
+
+## Upload all Azure Integration Account artifacts
+Azure Integration Accounts are critical for business-to-business (B2B) and integration workflows with Azure Logic Apps. The communication between different parties often requires artifacts. These artifacts include [trading partners](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners), [agreements](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-agreements), [maps](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [schemas](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [certificates](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-certificates)...
```
Comment (pim-simons) [inline community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md:5]:
```suggestion
Azure Integration Accounts are critical for business-to-business (B2B) and integration workflows with Azure Logic Apps. The communication between different parties often requires artifacts. These artifacts include [trading partners](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners), [agreements](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-agreements), [maps](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-schemas), [schemas](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [certificates](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-certificates)...
```

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+When such an Azure Integration Account was previously setup in a client project, this was done manually in the Azure portal or via custom scripts. This tedious repeated work was the perfect candidate to be included in the Arcus Scripting library.
+
+Starting from v0.6, one can finally upload all these artifacts with clean, well-named PowerShell functions. This wil reduce the custom scripts being copied from project to project, increase support and maintenance, and make the DevOps code a lot cleaner.
```
Comment (pim-simons) [inline community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md:9]:
```suggestion
Starting from v0.6, one can finally upload all these artifacts with clean, well-named PowerShell functions. This wil reduce the custom scripts being copied from project to project, improve support and maintenance, and make the DevOps code a lot cleaner.
```

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+# ----------
+# Uploading certificate 'MySecondCertificate.cer' into the Azure Integration Account 'my-integration-account'
+# Certificate 'MySecondCertificate.cer' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+```
+
+For more information on all things Azure Integration Account, see our [documentation page](https://scripting.arcus-azure.net/Features/powershell/azure-integration-account) where all the available PowerShell functions are described.
+
+## Adding Azure App Service application settings with ease
+Azure App Services can have common application settings. Just like local application settings, these should not contain any sensitive or secret information, but are especially useful to pass-in dynamically configuration values that can manipulate the application. Setting such application settings has to be done via the Azure portal. With Arcus Scripting v0.6, you don't have to make that trip to the portal anymore.
```
Comment (pim-simons) [inline community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md:39]:
```suggestion
Azure App Services can have common application settings. Just like local application settings, these should not contain any sensitive or secret information, but are especially useful to pass-in dynamic configuration values that can manipulate the application. Setting such application settings has to be done via the Azure portal. With Arcus Scripting v0.6, you don't have to make that trip to the portal anymore.
```

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+# App service has been found
+# Extracting the existing application settings
+# Setting the application setting 'LogLevel'
+# Successfully set the application setting 'LogLevel' of the App Service 'my-app-service' within resource group 'my-resource-group'
+```
+
+For more information on this module, see our [documentation page](https://scripting.arcus-azure.net/Features/powershell/azure-app-service).
+
+## Extended ARM template content injection with absolute file paths
+One of the new features in v0.6 is entirely written by a contributor. It's great to see our library expanding with features that were missing. All these additions grow from the usage of the library and the healthy reflex to contribute the missing pieces. In this case, our ARM template functionality to inject content into the template was missing something. We allowed relative value files to be injected into the ARM template but didn't had support for absolute external file paths. The way it was updated made sure that we stay backwards compatible with both relative file paths, making this a very flexible piece of functionality.
```
Comment (pim-simons) [inline community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md:67]:
```suggestion
One of the new features in v0.6 is entirely written by a contributor. It's great to see our library expanding with features that were missing. All these additions grow from the usage of the library and the healthy reflex to contribute the missing pieces. In this case, our ARM template functionality to inject content into the template was missing something. We allowed relative value files to be injected into the ARM template but didn't have support for absolute external file paths. The way it was updated made sure that we stay backwards compatible with both relative file paths, making this a very flexible piece of functionality.
```

**Review by pim-simons — CHANGES_REQUESTED** [review]
Wow, really great blogpost, thanks for your time and effort placed into this! 🥇

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md (hunk 1 above)
Comment (stijnmoreels) [reply community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md:5]: Oh, I see, it's a link that's wrong, but an other one that's proposed here 😄 .

**Comment by stijnmoreels** [comment]
> Wow, really great blogpost, thanks for your time and effort placed into this! 🥇

Thanks, Pim! Great review, nice to know that I can always count on you for a 'spelling sweep' 😉 .

File: community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md (hunk 1 above)
Comment (pim-simons) [reply community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md]: ah my bad

**Review by pim-simons — APPROVED** [review]

**Comment by fgheysels** [comment]
Nice blogpost.  Not too long and informative, I learned some new features of Arcus scrpting :)

**Comment by stijnmoreels** [comment]
Great! Thx all for reviewing!

**Pull request merged**